handler/mosn/filter.go

```diff
@@ -0,0 +1,189 @@
+package mosn
+
+import (
+	"context"
+	"errors"
+	"os"
+	"runtime"
+	"strconv"
+
+	"mosn.io/api"
+	"mosn.io/mosn/pkg/log"
+
+	httpwasm "github.com/http-wasm/http-wasm-host-go"
+	"github.com/http-wasm/http-wasm-host-go/api/handler"
+	internalhandler "github.com/http-wasm/http-wasm-host-go/internal/handler"
+)
+
+func init() {
+	api.RegisterStream("httpwasm", factoryCreator)
```

Comment:
is this right?

```suggestion
	// There's no API to configure a StreamFilter without using the global registry.
	api.RegisterStream("httpwasm", factoryCreator)
```

Reply:
Yup, nor many components like HTTP/1 support it seems

https://github.com/http-wasm/http-wasm-host-go/pull/19/files#diff-70c33dc0052ff54bdbe5db4e36ad9699fc1205abbf5eb0764fa05214771398f6R19